recipes/underworld/conda_build_config.yaml

```diff
@@ -0,0 +1,5 @@
+mpi:
+  - mpich
+  #- openmpi [linux]
+petsc:
```

Comment:
```suggestion
# Required until PETSC migration to 3.18 is complete.
petsc:
```

Can you confirm that you will *not* compile successfully with petsc 1.17? otherwise, it is likely best for the migrator to come around.

Reply:
Yes we need >=3.18